examples/nginx-app/with-pv.yaml

```diff
@@ -43,7 +43,7 @@ metadata:
   name: nginx-deployment
   namespace: nginx-example
 spec:
-  replicas: 2
+  replicas: 1
```

Comment:
> It might be clearer to also update the snapshot example section to either specify --snapshot-volumes=true and --restore-volumes=true, or remove the flags entirely since its true by default

Let's remove them.